lib/Parameter.php

```diff
@@ -160,7 +160,7 @@ public static function guessParameterNameByValue(string $value): string
      *
      * This may be either a single, or multiple strings in an array.
      *
-     * @param string|array $value
+     * @param int|string|array $value
```

Comment:
is it possible to define more precise array-key and value types?

Reply:
probably - I will look in more detail when I get to do a lot a "real code".